text/20_requirements.md

```diff
@@ -108,7 +108,7 @@ When applied to physical phenomena, it is manifestly clear how silly it would be
 
 ## Functional versus Non-Functional
 
-Functional requirements state what a system should _do_; non-functional requirements state what a system should _be_.
+__Functional requirements__ state what a system should _do_; __non-functional requirements__ state what a system should _be_.
```

Comment:
Bolded italics are used specifically for "the first time a new term is introduced".  I may want to mention that somewhere.